src/ts/alert.ts

```diff
@@ -310,12 +322,33 @@ class AlertStatic {
         return this.crmContext;
     }
 
+    public setButtonEnabled(
+        internalId: string,
+        isEnabled: boolean,
+        childCall: boolean
+    ): void {
+        if (!internalId) {
+            alert("internalId parameter is not specified.");
+        }
+
+        const button = childCall ? parent.jQuery("#alertJs-Button-" + internalId) : this.$("#alertJs-Button-" + internalId);
+        button.prop("disabled", !isEnabled);
+    }
+
     // Use the returned iframe context with jQuery to get data from the iframe, i.e. this.$("#something", Alert.getIFrameContext().document);
-    public getIFrameWindow(): Window | undefined {
+    public getIFrameWindow(subgridCall: boolean): Window | undefined {
         let iframeContext: Window | null = null;
 
+        if (subgridCall) {
```

Comment:
The same code is run two times. But in the first case without checking of `isInitialized` flag.